packages/interactivity-router/src/index.ts

```diff
@@ -90,11 +96,7 @@ const fetchPage = async ( url: string, { html }: { html: string } ) => {
 const regionsToVdom: RegionsToVdom = ( dom, { vdom, url } = {} ) => {
 	const regions = { body: undefined };
 	const styles = prepareStyles( dom, url );
-	const scriptModules = [
-		...dom.querySelectorAll< HTMLScriptElement >(
-			'script[type=module][src]'
-		),
-	].map( ( s ) => s.src );
+	const scriptModules = preloadModules( dom );
```

Comment:
Not a big deal, but shouldn't these two methods have consistent names?

```suggestion
	const styles = preloadStyles( dom, url );
	const scriptModules = preloadModules( dom );
```